### PR TITLE
[6.4] Update beats framework to 151c55e

### PR DESCRIPTION
**Pull request by graphaelli**

### CODE REVIEW

**Comment by graphaelli** [comment]
jenkins, retest this please

File: vendor/vendor.json [hunk 1]
```diff
@@ -361,1218 +361,1248 @@
...
 			"path": "github.com/elastic/beats/libbeat/common/jsontransform",
-			"revision": "55e2c0d570b049eacaa56e1d251e8cdef9af8352",
-			"revisionTime": "2018-10-02T15:06:00Z",
+			"revision": "151c55e3ced102b4056e2a3f62acc82e288d8368",
+			"revisionTime": "2018-10-17T14:53:47Z",
+			"version": "6.4",
+			"versionExact": "6.4"
+		},
+		{
+			"path": "github.com/elastic/beats/libbeat/common/kafka",
```
Comment (simitt) [inline vendor/vendor.json]: This entry looks a bit weird, no `checksumSHA1` and no `revisionTime`

File: vendor/vendor.json [hunk 2]
```diff
@@ -361,1218 +361,1248 @@
...
-			"revision": "55e2c0d570b049eacaa56e1d251e8cdef9af8352",
-			"revisionTime": "2018-10-02T15:06:00Z",
+			"revision": "151c55e3ced102b4056e2a3f62acc82e288d8368",
+			"revisionTime": "2018-10-17T14:53:47Z",
+			"version": "6.4",
+			"versionExact": "6.4"
+		},
+		{
+			"path": "github.com/elastic/beats/libbeat/common/reload",
+			"revision": "6.4",
```
Comment (simitt) [inline vendor/vendor.json]: same comment as above, entry looks not quite right

File: vendor/vendor.json [hunk 3]
```diff
@@ -361,1218 +361,1248 @@
...
-			"revision": "55e2c0d570b049eacaa56e1d251e8cdef9af8352",
-			"revisionTime": "2018-10-02T15:06:00Z",
+			"revision": "151c55e3ced102b4056e2a3f62acc82e288d8368",
+			"revisionTime": "2018-10-17T14:53:47Z",
+			"version": "6.4",
+			"versionExact": "6.4"
+		},
+		{
+			"path": "github.com/elastic/beats/libbeat/management",
+			"revision": "6.4",
```
Comment (simitt) [inline vendor/vendor.json]: again

File: vendor/vendor.json [hunk 4]
```diff
@@ -361,1218 +361,1248 @@
...
-			"revision": "55e2c0d570b049eacaa56e1d251e8cdef9af8352",
-			"revisionTime": "2018-10-02T15:06:00Z",
+			"revision": "151c55e3ced102b4056e2a3f62acc82e288d8368",
+			"revisionTime": "2018-10-17T14:53:47Z",
+			"version": "6.4",
+			"versionExact": "6.4"
+		},
+		{
+			"path": "github.com/elastic/beats/libbeat/processors/add_process_metadata",
+			"revision": "6.4",
```
Comment (simitt) [inline vendor/vendor.json]: same as above

File: vendor/vendor.json [hunk 5]
```diff
@@ -361,1218 +361,1248 @@
...
-			"revision": "55e2c0d570b049eacaa56e1d251e8cdef9af8352",
-			"revisionTime": "2018-10-02T15:06:00Z",
+			"revision": "151c55e3ced102b4056e2a3f62acc82e288d8368",
+			"revisionTime": "2018-10-17T14:53:47Z",
+			"version": "6.4",
+			"versionExact": "6.4"
+		},
+		{
+			"path": "github.com/elastic/beats/libbeat/processors/dns",
+			"revision": "6.4",
```
Comment (simitt) [inline vendor/vendor.json]: same as above

**Pull request merged**